easybuild/framework/easyconfig/default.py

```diff
@@ -155,12 +155,13 @@
     'exts_list': [[], 'List with extensions added to the base installation', EXTENSIONS],
 
     # MODULES easyconfig parameters
+    'modaliases': [{}, "Aliases to be defined in module file", MODULES],
     'modextrapaths': [{}, "Extra paths to be prepended in module file", MODULES],
     'modextravars': [{}, "Extra environment variables to be added to module file", MODULES],
     'modloadmsg': [{}, "Message that should be printed when generated module is loaded", MODULES],
     'modluafooter': ["", "Footer to include in generated module file (Lua syntax)", MODULES],
+    'modname': [None, "Module name to use (rather than using software name", MODULES],
```

Comment:
@rjeschmi: quoting you: "It is the "name" in the modulefile", so `name_in_module` is almost a one-to-one match?

'meta' is so... meta. ;-)